deeppavlov/models/elmo/elmo2tfhub.py

```diff
@@ -198,6 +207,8 @@ def export2hub(weight_file, hub_dir, options):
 
             with tf.Session() as sess:
                 sess.run(tf.global_variables_initializer())
+                if hub_dir.exists():
+                    rmdir(hub_dir)
```

Comment:
https://docs.python.org/3/library/shutil.html#shutil.rmtree

Reply:
it's updated